experimental/dds/tree2/src/core/tree/persistedTreeTextFormat.ts

```diff
@@ -79,7 +79,7 @@ interface EncodedGenericTreeNode<TChild>
 	extends EncodedGenericFieldsNode<TChild>,
 		EncodedNodeData {}
 const EncodedGenericTreeNode = <Schema extends TSchema>(tChild: Schema) =>
-	Type.Intersect([EncodedGenericFieldsNode(tChild), EncodedNodeData], {
+	Type.Composite([EncodedGenericFieldsNode(tChild), EncodedNodeData], {
```

Comment:
See https://github.com/microsoft/FluidFramework/pull/18519#discussion_r1408573941. TL:DR: `Type.Composite` combines the requirements at schema declaration time, `Type.Intersect` combines them at schema validation time, so to use `noAdditionalProperties` you generally want Composite.